src/main/webapp/i18n/de/programmingAssessment.json

```diff
@@ -5,7 +5,13 @@
                 "heading": "Einreichungen und Bewertungen der Textaufgabe \"{{exerciseTitle}}\""
             },
             "confirmCancel": "Bist du sicher, dass du die Bewertung abbrechen möchtest? Dein aktueller Bewertungsfortschritt wird gelöscht und die Abgabe steht anderen Tutor:innen wieder zur Verfügung.",
-            "notFound": "Wir haben keine neue Programmier Abgabe ohne Bewertung gefunden, bitte gehe zurück."
+            "notFound": "Wir haben keine neue Programmier Abgabe ohne Bewertung gefunden, bitte gehe zurück.",
+            "penalty": "Penalty",
+            "issues": "Issues",
```

Comment:
IIRC in other places those non-translations are used as well. ‘Punktabzug’ and ‘Probleme’ would probably be better.

(Just a note so I don’t forget about it, should probably be changed consistently everywhere in a separate PR.)

Reply:
I agree that a translation would be better. I decided to stick with the English terms to keep it consistent to the explanation text above the chart. There, the terms stay in English as well.